nbbuild/javadoctools/template.xml

```diff
@@ -280,23 +279,69 @@ cause it to fail.
         <!-- Timestamp is handled in javadoc.css to avoid too many commits on generated HTML files. -->
         <property name="javadoc.footer" value='&lt;span class="footnote"&gt;&lt;/span&gt;&lt;!-- See javadoc.css for timestamp information --&gt;'/>
     </target>
-    
-    <target name="javadoc-exec-packages" depends="javadoc-init,javadoc-generate-references,javadoc-generate-overview,javadoc-exec-condition,javadoc-check-timestamps,javadoc-make-plain-title,javadoc-make-hyperlinked-title,javadoc-exec-condition,-javadoc-set-footer" unless="javadoc.exec.packages">
-        <javadoc source="${javac.source}" author="false" destdir="${javadoc.out.dir}" packagenames="${javadoc.packages}" stylesheetfile="${javadoc.css.main}" windowtitle="${javadoc.title}" overview="${javadoc.overview}" splitindex="true" use="true" version="false" useexternalfile="true" encoding="UTF-8">
+
+
+    <target name="-javadoc-exec-packages-nbjavac" depends="javadoc-init,javadoc-generate-references,javadoc-generate-overview,javadoc-exec-condition,javadoc-check-timestamps,javadoc-make-plain-title,javadoc-make-hyperlinked-title,javadoc-exec-condition,-javadoc-set-footer" if="javadoc.nbjavac">
```

Comment:
-javadoc-exec-packages-nbjavac target only active javadoc.nbjavac
currently as mentioned only 2 modules need that. javadoc in this case not fail if error.
it's possible to test on module java.source.base. ant clean;ant build; ant javadoc.

As this target is not working cannot prepend the bootclasspath with nb-javac-jdk this lead to this module not visible in the apidoc (except their arch,apichanges)